### ISSUE TITLE
Bugs and features

### ISSUE THREAD

**Issue by hugobruna** [bytefury/crater#271]
Hello,

I am using a crater on a hosting provider's server and I have found some bugs : 

- When you modify something like a user, an invoice or an item ... an unknown error appears after submit (like a notification).

Concerning features : 

- Translate on pdf the data (if you want, I can translate into French the information generated by the pdf in a JSON file like the user interface translation) 
- Add a system to customise the colour of UI and pdf (total, business name if you don't have a logo)

Regards 👍

**Comment by mohitpanjwani** [comment]
Hey, thanks for the feedback. 

Can you change `APP_DEBUG=true` in `.env` file and then paste the screenshot of the error that you're receiving on estimate pdf and other pages?

**Comment by hugobruna** [comment]
Hello again,

I made a mistake on my code, I replaced the variable "$estimate" by "$invoice" in the view (blade) > partial part for the pdf.
There are no bugs with pdf. I didn't pay attention, sorry for that.

Concerning the other bugs, look on the screenshot. It's the same error for all the things I edit. There are no logs on these errors.

![Annotation 2020-09-23 173238](https://user-images.githubusercontent.com/38312920/94042903-a6f4e380-fdbb-11ea-8dd4-1897b44efa14.png)

Regards,
